examples/furniture.py

```diff
@@ -14,14 +14,14 @@
     [1, 2],  # Lathe
     [3, 1.5],  # Polisher
 ]
-activity = makeDict([Resources, Chairs], activity)
+activity_dict = makeDict([Resources, Chairs], activity)
```

Comment:
I'm curious, is this some kind of naming convention to recognize the type objects? Why rename all dictionaries to have a `_dict` at the end?

Reply:
I changed the name of this variable because previously this line was changing the type of `activity` from a list to a dict. The `_dict` is a bit lazy admittedly. Very open to better name suggestions :)